src/main/java/com/opencart/steps/LoginPageBL.java

```diff
@@ -0,0 +1,49 @@
+package com.opencart.steps;
+
+
+import com.opencart.datamodel.LoginModel;
+import com.opencart.pages.LoginPage;
+import com.opencart.pages.SuccessLoginPage;
+import com.opencart.repository.LoginModelRepository;
+import org.testng.Assert;
+
+public class LoginPageBL {
+    private LoginPage loginPage;
+    private SuccessLoginPage successLoginPage;
+
+    public LoginPageBL() {
+        loginPage = new LoginPage();
+    }
+
+    public LoginPageBL loginPerson() {
+        LoginModel loginModel = LoginModelRepository.getLoginModel();
```

Comment:
It is okay, that you have created LoginModelRepository. But it is better to use it when you will need to test some flow,  by the logged user. So, as not to create a new user every time, you will store it within the repository, Also you can add some invalid models there, for verifying more Login cases. In a particular case, try to register a new user, then log-out and then log in, by that user.